app/create/components/Form.tsx

```diff
@@ -0,0 +1,100 @@
+'use client';
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
+
+import * as z from 'zod';
+function Form() {
+  const [file, setFile] = useState('');
+  const schema = z.object({
```

Comment:
1. it's better to separate the logic from the UI, for example, the Zod schema and API calls can be put in other parts of the code  .
2. zod schema doesn't have the pictureUrl field in it 
3. In the validateForm , if the request fails , it should show the user some feedback on what went wrong
4. some inputs are using setvalue and some a registered to zod , better to be consistent